docs/settings/settings.md

```diff
@@ -119,34 +132,63 @@ A Speech To Text(STT) is an engine used to translate what you say into a text th
 By default, Kalliope uses google STT engine.
 
 The following syntax is used to provide the engine name:
-```yml
+```yaml
 default_speech_to_text: "stt_name"
 ```
 
 E.g
-```yml
+```yaml
 default_speech_to_text: "google"
 ```
 
-Get the full list of [SST engine here](stt.md).
-
 ### speech_to_text
 Each STT has it own configuration. This configuration is passed as argument as shown bellow
-```yml
+```yaml
 speech_to_text:
   - stt_name:
       parameter_name: "value"
 ```
 
 E.g:
-```yml
+```yaml
 speech_to_text:
   - google:
       language: "fr-FR"
   - bing
 ```
 
-Some arguments are required, some others are optional, please refer to the [STT documentation](stt.md) to know available parameters for each supported STT.
+### Settings example
+
+```yml
+default_speech_to_text: "google"
+speech_to_text:
+  - google:
+      language: "fr-FR"
+  - wit:
+      key: "B5JI3YUSLYOYWNIZRNBVM34XUODME2K"
```

Comment:
Those keys are random ? if no, I think we should not keep real key in the doc

Reply:
yep it's random keys for all